lib/src/feature/initialization/data/platform/platform_initialization_js.dart

```diff
@@ -1,35 +1,30 @@
 // ignore_for_file: avoid_web_libraries_in_flutter
 
-import 'dart:html' as html;
-
-//import 'package:flutter_web_plugins/url_strategy.dart';
-//import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'package:web/web.dart';
 
 Future<void> $platformInitialization() async {
-  //setUrlStrategy(const HashUrlStrategy());
+  // setUrlStrategy(const HashUrlStrategy());
 
   // Remove splash screen
   Future<void>.delayed(
     const Duration(seconds: 1),
     () {
       // Before running your app:
-      //setUrlStrategy(null); // const HashUrlStrategy();
-      //setUrlStrategy(NoHistoryUrlStrategy());
+      // setUrlStrategy(null); // const HashUrlStrategy();
+      // setUrlStrategy(NoHistoryUrlStrategy());
+
+      document.getElementById('splash')?.remove();
+      document.getElementById('splash-branding')?.remove();
+      document.body?.style.background = 'transparent';
 
-      html.document.getElementById('splash')?.remove();
-      html.document.getElementById('splash-branding')?.remove();
-      html.document.body?.style.background = 'transparent';
-      html.document
-          .getElementsByClassName('splash-loading')
-          .toList(growable: false)
-          .forEach((element) => element.remove());
+      final elements = document.getElementsByClassName('splash-loading');
+      for (var i = 0; i < elements.length; i++) elements.item(i)?.remove();
```

Comment:
Thats potentially can be unsafe to remove it that way
You should prefer reverse itteration